ports/stm32/boards/NUCLEO_H743ZI/mpconfigboard.h

```diff
@@ -19,26 +26,68 @@ void NUCLEO_H743ZI_board_early_init(void);
 #define MICROPY_HW_CLK_PLLQ (4)
 #define MICROPY_HW_CLK_PLLR (2)
 
+// with the 8MHz HSE, settings of the PLL for USB
+#define MICROPY_HW_PLL3_PLL3M (4)
+#define MICROPY_HW_PLL3_PLL3N (120)
+#define MICROPY_HW_PLL3_PLL3P (2)
+#define MICROPY_HW_PLL3_PLL3Q (5)
+#define MICROPY_HW_PLL3_PLL3R (2)
+
 // 4 wait states
 #define MICROPY_HW_FLASH_LATENCY    FLASH_LATENCY_4
 
-// UART config
-#define MICROPY_HW_UART3_TX         (pin_D8)
-#define MICROPY_HW_UART3_RX         (pin_D9)
+// alternative REPL port
 #define MICROPY_HW_UART_REPL        PYB_UART_3
 #define MICROPY_HW_UART_REPL_BAUD   115200
 
-// I2C busses
-#define MICROPY_HW_I2C1_SCL         (pin_B8)
-#define MICROPY_HW_I2C1_SDA         (pin_B9)
-#define MICROPY_HW_I2C2_SCL         (pin_F1)
-#define MICROPY_HW_I2C2_SDA         (pin_F0)
+// UART1 config
+#define MICROPY_HW_UART1_TX   (pin_B6)
+#define MICROPY_HW_UART1_RX   (pin_B15)
+
+// UART2 config
+#define MICROPY_HW_UART2_TX   (pin_D5)
+#define MICROPY_HW_UART2_RX   (pin_D6)
+#define MICROPY_HW_UART2_CTS  (pin_A0) // optional use
+#define MICROPY_HW_UART2_RTS  (pin_D4) // optional use
+
+// UART3 config
+#define MICROPY_HW_UART3_TX    (pin_D8)  // STLINK VCOM
+#define MICROPY_HW_UART3_RX    (pin_D9)  // STLINK VCOM
+//#define MICROPY_HW_UART3_CTS    (pin_D11) // optional use
+//#define MICROPY_HW_UART3_RTS    (pin_D12) // optional use
+
+// UART6 Available serial interface on Arduino connector
+#define MICROPY_HW_UART6_TX   (pin_G14) // "D1"
+#define MICROPY_HW_UART6_RX   (pin_G9)  // "D0"
+#define MICROPY_HW_UART6_CTS  (pin_G15) // optional use
+#define MICROPY_HW_UART6_RTS  (pin_G8)  // optional use
+
+// UART7 config
+#define MICROPY_HW_UART7_TX    (pin_E8)
+#define MICROPY_HW_UART7_RX    (pin_E7)
+#define MICROPY_HW_UART7_CTS  (pin_E10) // optional use
+#define MICROPY_HW_UART7_RTS  (pin_E9)  // optional use
+
+// UART8 config
+#define MICROPY_HW_UART8_TX    (pin_E1)
+#define MICROPY_HW_UART8_RX    (pin_E0)
+#define MICROPY_HW_UART8_CTS  (pin_D14) // optional use
+#define MICROPY_HW_UART8_RTS  (pin_D15) // optional use
+
+
+// Available I2C interface on Arduino connector
+#define MICROPY_HW_I2C1_SCL    (pin_B8)  // "D15"
+#define MICROPY_HW_I2C1_SDA    (pin_B9)  // "D14"
+
+#define MICROPY_HW_I2C2_SCL    (pin_F1)
+#define MICROPY_HW_I2C2_SDA    (pin_F0)
+// optional when using SMBUS instead of I2C
+#define MICROPY_HW_I2C2_SMB    (pin_F2)  // not implemented
```

Comment:
There's currently no such configuration called I2C_SMB so please remove this line to avoid confusion.

Reply:
I will do so